cpu/stm32l1/stmclk.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_stm32l1
+ * @{
+ *
+ * @file
+ * @brief       Implementation of the kernel cpu functions
```

Comment:
I think here something like `Implemetation of CPU clock functions` might be more suitable.